pyproject.toml

```diff
@@ -91,7 +91,7 @@ fmt         = ["black . --extend-exclude 'tests/unit/source_code/samples/*'",
                "ruff check . --fix",
                "mypy --disable-error-code 'annotation-unchecked' --exclude 'tests/unit/source_code/samples/*' .",
                "pylint --output-format=colorized -j 0 src tests"]
-verify      = ["black --check .",
+verify      = ["black --check . --extend-exclude 'tests/unit/source_code/samples/*'",
```

Comment:
Out-of-scope of this PR

Reply:
@JCZuurmond https://github.com/databrickslabs/ucx/pull/1742